examples/humble/go/views/app.go

```diff
@@ -107,8 +107,11 @@ func (v *App) CreateTodo(ev dom.Event) {
 	if !ok {
 		panic("Could not convert event target to dom.HTMLInputElement")
 	}
-	v.Todos.AddTodo(strings.TrimSpace(input.Value))
-	document.QuerySelector(".new-todo").(dom.HTMLElement).Focus()
+	title := strings.TrimSpace(input.Value)
+	if title != "" {
```

Comment:
Optional style suggestion. Since `title` is only needed inside the if block, have you considered writing it this way?

```Go
if title := strings.TrimSpace(input.Value); title != "" {
```